src/cif.cc

```diff
@@ -85,16 +86,13 @@ structure_from_doc(const gemmi::cif::Document &doc,
             }
 
             // Pick the first alternative conformation for an atom
-            if ((currentAltId != '?' && prevAltId == '?') || (currentAltId == '?')) {
-                prevAltId = currentAltId;
-            } else if (currentAltId != '?' && currentAltId != prevAltId) {
+            if (currentAltId != '.' && currentAltId != 'A') {
```

Comment:
Are we sure alt ids always begin with 'A'? (If so, that is a nice simplification that should be used in the pdb code as well)

Reply:
So Alt ID is either a `.` if there is only one rotamer/conformer for that residue. 

If there are several they are distinguished by A, B, C, etc. And in our case we default to the first one, which is A.

I hope this is enough. This is where that test set of pdb/cif files comes in handy.

Reply:
Right, the other solution just takes the first one, even if A is missing for some reason. But I guess we'll find out in our tests as you say. This is cleaner if it works.